ui.tests/test-module/specs/actions/customProperties/customProperties.spec.js

```diff
@@ -0,0 +1,104 @@
+const afConstants = require("../../../libs/commons/formsConstants");
+const sitesSelectors = require("../../../libs/commons/sitesSelectors");
+/*******************************************************************************
+ * Copyright 2023 Adobe
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ******************************************************************************/
+describe("Custom Properties Tests", () => {
+
+    const dropTextInputInContainer = function () {
+        const dataPath = "/content/forms/af/core-components-it/blank/jcr:content/guideContainer/*",
+            responsiveGridDropZoneSelector = sitesSelectors.overlays.overlay.component + "[data-path='" + dataPath + "']";
+        cy.selectLayer("Edit");
+        cy.insertComponent(responsiveGridDropZoneSelector, "Adaptive Form Text Box", afConstants.components.forms.resourceType.formtextinput);
+        cy.get('body').click(0, 0);
+    };
+
+    context('Open Forms Editor', function () {
+        const pagePath = "/content/forms/af/core-components-it/blank",
+            textInputEditPath = pagePath + afConstants.FORM_EDITOR_FORM_CONTAINER_SUFFIX + "/textinput",
+            textInputEditPathSelector = "[data-path='" + textInputEditPath + "']",
+            textInputDrop = pagePath + afConstants.FORM_EDITOR_FORM_CONTAINER_SUFFIX + "/" + afConstants.components.forms.resourceType.formtextinput.split("/").pop();
+
+        it('End to End Custom Property Test', function () {
+            const textInputEditBoxSelector = "coral-tag[value='/conf/core-components-examples/settings/wcm/templates/af-blank-v2/structure/jcr:content/guideContainer/forms-components-examples/components/form/textinput']";
+            const tabSelector = '._coral-Tabs--horizontal ._coral-Tabs-item';
+            const submitBtnSelector = ".cq-dialog-submit";
+            const AFv2FormTemplatePath = "/conf/core-components-examples/settings/wcm/templates/af-blank-v2/structure.html";
+            // Exceptions coming from coral and template policy
+            cy.on('uncaught:exception', () => {
+                return false
+            });
+            // open template policy
+            cy.openAFv2TemplateEditor();
+            cy.get(textInputEditBoxSelector).find("button").click({force: true});
+            cy.get('.cq-dialog').should('be.visible');
+            cy.get(".coral-Form-fieldwrapper").contains("Policy Title *").next().focus().clear().type("Test policy");
```

Comment:
Should we use .coral css classes for selectors? aren't these private css class
@rismehta